libs/explored-js/README.md

```diff
@@ -1,6 +1,6 @@
 # explored-js
 
-SDK for interacting with `explored`.
+Provides an axios-based typescript sdk for interacting with the [`explored`](https://github.com/SiaFoundation/explored) api.
```

Comment:
I changed the text here but I did not include additional info about axios config because I'm unsure about it/where I'd put it. If you have ideas for this, feel free to suggest or even change it after the fact. Whatever works.

Reply:
Dont worry about it now, but in case we want to in the future, what I was thinking is you could add an "Axios" section with an example showing that you can pass axios options, renterd-js has one, eg:
```
  await worker.objectUpload({
    params: {
      key: 'path/to/file.txt',
      bucket: 'my-bucket',
    },
    data: new File(['file contents'], 'file.txt'),
    config: {
      onUploadProgress: (progress) => {
        console.log(progress.loaded / progress.total)
      },
    },
  })
  ```